### PR TITLE
completed array3 exercises

### PR DESCRIPTION
**Pull request by goelvatsal**

### CODE REVIEW

File: 14-arrays/exercises/11-average/main.go [hunk 1]
```diff
@@ -40,4 +46,28 @@ package main
...
+
+	var (
+		arr     [5]int
+		avg     int
+		lenArgs int
+	)
+
+	for i := 1; i < len(os.Args); i++ {
+		n, err := strconv.Atoi(os.Args[i])
+		if err != nil {
```
Comment (goel4607) [inline 14-arrays/exercises/11-average/main.go]: simplify the code here as discussed

Comment (goelvatsal) [reply 14-arrays/exercises/11-average/main.go]: fixed

File: 14-arrays/exercises/11-average/main.go [hunk 2]
```diff
@@ -59,9 +59,7 @@ func main() {
 
 	for i := 1; i < len(os.Args); i++ {
 		n, err := strconv.Atoi(os.Args[i])
-		if err != nil {
-			continue
-		} else {
+		if err == nil {
 			lenArgs += 1
 		}
 		arr[i-1] = n
```
Comment (goel4607) [inline 14-arrays/exercises/11-average/main.go:66]: here avg should be updated within if statement

Comment (goelvatsal) [reply 14-arrays/exercises/11-average/main.go:66]: fixed

**Pull request merged**